src/main/java/com/powsybl/openloadflow/sensi/AcSensitivityAnalysis.java

```diff
@@ -205,6 +213,24 @@ public void analyse(Network network, List<PropagatedContingency> contingencies,
                 slackParticipationByBus = Collections.singletonMap(lfNetwork.getSlackBus(), -1d);
             }
 
+            // if we have at least one bus target voltage linked to a ratio tap changer, we have to rebuild the AC equation
+            // system obtained just before the transformer steps rounding.
+            if (hasTransformerBusTargetVoltage) {
+                for (OuterLoop outerLoop : engine.getParameters().getOuterLoops()) {
+                    if (outerLoop instanceof TransformerVoltageControlOuterLoop) {
+                        outerLoop.setActive(false);
+                    }
+                }
+                for (LfBus bus : lfNetwork.getBuses()) {
+                    if (bus.getDiscreteVoltageControl() != null && bus.getDiscreteVoltageControl().getMode().equals(DiscreteVoltageControl.Mode.OFF)) {
+                        // switch on regulating transformers
+                        bus.getDiscreteVoltageControl().setMode(DiscreteVoltageControl.Mode.VOLTAGE);
+                    }
+                }
+                engine.setEquationSystem(null); //FIXME: should be an update only.
```

Comment:
@geofjamg it could be improved here: an update is enough using the `AcEquationSystemUpdater` but I don't know how to do.